servicesContracts.py

```diff
@@ -1,28 +1,31 @@
 from typing import Any
-from servicesContracts.requestEthereum_service import RequestEthereumService
```

Comment:
I think you still need to import the `RequestEthereumService`

Reply:
Yes, but I removed it because It gives a syntax error. The code in requestEthereum_service  is still JS code.